### PR TITLE
Restore `edit-readme` in "Repository refresh" beta

### PR DESCRIPTION
**Pull request by FloEdelmann**
Relates to #3081.

### CODE REVIEW

File: source/features/edit-readme.tsx [hunk 1]
```diff
@@ -13,9 +13,10 @@ async function init(): Promise<void | false> {
 		return false;
 	}
 
-	const isPermalink = /Tag|Tree/.test(select('.branch-select-menu i')!.textContent!);
+	const isPermalink = /Tag|Tree/.test(select('.branch-select-menu i, #branch-select-menu i')!.textContent!);
 	const filename = readmeHeader.textContent!.trim();
-	const fileLink = select<HTMLAnchorElement>(`.files [title="${filename}"]`)!;
+	const filesList = select('.files, #files ~ .Details')!;
+	const fileLink = select<HTMLAnchorElement>(`[title="${filename}"]`, filesList)!;
```
Comment (fregante) [inline source/features/edit-readme.tsx:19]: I think we can simplify this to
```suggestion
	const fileLink = select<HTMLAnchorElement>(`.js-navigation-open[title="${filename}"]`)!;
```

File: source/features/edit-readme.tsx [hunk 2]
```diff
@@ -13,9 +13,10 @@ async function init(): Promise<void | false> {
 		return false;
 	}
 
-	const isPermalink = /Tag|Tree/.test(select('.branch-select-menu i')!.textContent!);
+	const isPermalink = /Tag|Tree/.test(select('.branch-select-menu i, #branch-select-menu i')!.textContent!);
```
Comment (fregante) [inline source/features/edit-readme.tsx:16]:
```suggestion
	const isPermalink = /Tag|Tree/.test(select('[data-hotkey="w"] i')!.textContent!);
```

**Comment by fregante** [comment]
I also see this icon issue

# Refined GitHub



# Native

**Comment by FloEdelmann** [comment]
I think that's caused by `hide-readme-header`, I'm looking into it.

**Pull request merged**